### ISSUE TITLE
httpHeaders  implementation

### ISSUE THREAD

**Issue by megadrazz** [dkrockcom/cordova-plugin-locationbgservice#1]
url: 'http://10.0.0.109:3000/auth/location',
                 httpHeaders: {
                     "authorization": "Bearer "+ token
                 },

Congratulations on the project.

Can you do this implementation?

**Comment by dkrockcom** [comment]
Yes I can and thanks for the reporting, it will resolve in next release.
